java/client/src/main/java/glide/api/commands/ServerManagementClusterCommands.java

```diff
@@ -212,4 +212,44 @@ public interface ServerManagementClusterCommands {
      * }</pre>
      */
     CompletableFuture<String> configSet(Map<String, String> parameters, Route route);
+
+    /**
+     * Returns the server time. <br>
+     * The command will be routed to a random node.
+     *
+     * @see <a href="https://redis.io/commands/time/">redis.io</a> for details.
+     * @return The current server time as a <code>String[]</code> with two elements: A Unix timestamp
```

Comment:
Up to this point we used "array" instead of String[] should we change all commands to specify the type. Just for consistency.

Reply:
I noticed that.  That's also what the node client uses.  I'll switch back, but I wanted to avoid saying "array of Strings" because it's a little cumbersome.